flashinfer/cute_dsl/add_rmsnorm_fp4quant.py

```
@@ -167,6 +194,14 @@ class AddRMSNormFP4QuantKernel:
     Supports both NVFP4 (block_size=16) and MXFP4 (block_size=32) formats.
     """
 
+    # Offline sweep-tuned launch overrides.  Unlisted configurations use the
+    # general scale-block heuristic below.
+    # (sm_version, H, block_size, is_fp16):
+    #     (cluster_n, threads_per_row, num_threads)
+    _KNOB_LUT = {
+        (103, 1024, 16, False): (1, 64, 256),
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)